apistrap/aiohttp.py

```diff
@@ -278,7 +284,68 @@ def _extract_specs(self) -> None:
 
             self.spec.path(url, {route.method.lower(): self._extract_operation_spec(route)})
 
-        self._specs_extracted = True
+    def _parse_parameter_value(self, parameter: inspect.Parameter, value: str):
+        if parameter.annotation == inspect.Parameter.empty:
+            return value
+
+        if parameter.annotation == str or parameter.annotation == "str":
+            return value
+
+        if parameter.annotation == int or parameter.annotation == "int":
+            return int(value)
+
+        return value
+
+    def _process_route_parameters(self, route: AbstractRoute) -> None:
+        """
+        If necessary, wrap the route handler with a coroutine that accepts a AioHTTP request, extracts parameters to
+        satisfy the underlying handler and forwards them to the original handler.
+
+        :param route: the route whose handler should be wrapped
+        """
+
+        signature = inspect.signature(route.handler)
+
+        request_param: Optional[inspect.Parameter] = next(
+            filter(lambda p: issubclass(p.annotation, BaseRequest), signature.parameters.values()), None
```

Comment:
are we sure there's at most one request? do we want to assert that?

Reply:
That's probably a good idea.